lilypad/server/settings.py

```diff
@@ -18,19 +18,25 @@ class Settings(BaseSettings):
     # JWT settings
     jwt_secret: str = Field(default="my_secret_key", description="JWT secret key")
     jwt_algorithm: str = "HS256"
-    jwt_expiration_minutes: int = Field(
-        default=1440, description="Token expiration in minutes (24 hours)"
-    )
 
     environment: str = Field(default="development")
+    db_host: str | None = None
+    db_name: str | None = None
+    db_user: str | None = None
+    db_password: str | None = None
+    db_port: int | None = None
 
     @property
     def config(self) -> dict[str, Any]:
         """Get the configuration for the current environment"""
         configs = {
             "development": {
                 "api_url": "http://localhost:8000/api",
-                "client_url": "http://localhost:5173",
+                "client_url": "",
+            },
+            "local": {
+                "api_url": "http://localhost:8000/api",
```

Comment:
shouldn't the port be configurable?

Reply:
hmm yeah, i will add a new env that is backend port